src/api/admin.ts

```diff
@@ -0,0 +1,19 @@
+import express from 'express';
+
+export const admin = express.Router();
+
+admin.post('/login', (req, res): void => {
+  if (req.body.secret !== process.env.ADMIN_SECRET) {
```

Comment:
Can we move this to a `header`? That's what I've been doing with the other routes.

Reply:
This is a login request though, only for the front end. And usually you send form data through a post. It uses auth header for making calls later